StaticRouter missing

## Version
4.0.0-beta.2

## Steps to reproduce
npm install
build with webpack

## Result
`ERROR in ./~/react-router/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./StaticRouter in /<PROJECT>/node_modules/react-router
 @ ./~/react-router/index.js 26:21-46`

I looked at `node_modules/react-router/index.js` line 26 is `var _StaticRouter2 = require('./StaticRouter');` however StaticRouter.js is not in the `mode_modules/react-router` directory after `npm install`.

--- a/packages/react-router/package.json
+++ b/packages/react-router/package.json
@@ -9,13 +9,12 @@
     "Ryan Florence"
   ],
   "files": [
-    "Core.js",
     "MemoryRouter.js",
     "Prompt.js",
     "Redirect.js",
     "Route.js",
     "Router.js",
-    "ServerRouter.js",
+    "StaticRouter.js",
     "Switch.js",
     "index.js",
     "matchPath.js",

